RHDEVDOCS-3306: Document running image build tasks as unprivileged builds (#44733)
openshift/openshift-docs: cicd/pipelines/running-workload-as-user-namespaces-on-openshift-pipelines.adoc

:_content-type: ASSEMBLY
[id="running-workload-as-user-namespaces-on-openshift-pipelines"]
= Running workload as user namespaces on OpenShift Pipelines
include::_attributes/common-attributes.adoc[]
:context: running-workload-as-user-namespaces-on-openshift-pipelines

toc::[]

> While running OpenShift Pipelines as root on the container, the host can be possibly exposes and the process in that container may be able to mingle with other resources. To elimiate any such discrepancies, we can run the workload as a user namespace to make sure that our integration is safe & secure. There are some use-cases where we need the workflow to be run as root on the container but still be secure on the host.

Reviewer: ```suggestion
Running OpenShift Pipelines as root  on a container can expose the host and container processes to other resources. In some cases, to reduce this type of exposure, you can run some workloads in a user namespace instead. In other cases, you may still need to run the workload as root on the container but still be secure on the host.
```
In this suggestion, I've applied the following principles:
- Avoid "we." User perceive it as condescending. Rather, use the second-person "you."
- Avoid passive voice. 
- Apply minimalism to your sentence structure. Use simple sentence structures. Remove unnecessary words.

> While running OpenShift Pipelines as root on the container, the host can be possibly exposes and the process in that container may be able to mingle with other resources. To elimiate any such discrepancies, we can run the workload as a user namespace to make sure that our integration is safe & secure. There are some use-cases where we need the workflow to be run as root on the container but still be secure on the host.

Reviewer: ```suggestion
Running OpenShift Pipelines as root  on a container can expose the host and container processes to other resources. In some cases, to reduce this type of exposure, you can run some workloads in a user namespace instead. In other cases, you may still need to run the workload as root on the container but still be secure on the host.
```
In this suggestion, I've applied the following principles:
- Avoid "we." User perceive it as condescending. Rather, use the second-person "you."
- Avoid passive voice. 
- Apply minimalism to your sentence structure. Use simple sentence structures. Remove unnecessary words.

Reviewer: Would it make sense for you to review the rest of the PR for areas where you can apply these principles, rather than have me point them out?
